examples/ios/objc/Migration/Examples/Example_v0.h

```diff
@@ -0,0 +1,81 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2021 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+#ifdef SCHEMA_VERSION_0
+
+#import <Foundation/Foundation.h>
+#import <Realm/Realm.h>
+
+#pragma mark - Schema
+
+int schemaVersion = 0;
+
+@interface Person : RLMObject
+@property NSString *firstName;
+@property NSString *lastName;
+@property int age;
```

Comment:
`age` should be an `NSInteger`– objc boxed types nearly always take precedence over c types.